ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -589,11 +589,11 @@ if get_option('enable-mediapipe')
   mediapipe_incdir = include_directories(
     MEDIAPIPE_HOME,
     join_paths(MEDIAPIPE_HOME, 'bazel-bin'),
-    join_paths(MEDIAPIPE_HOME, 'bazel-mediapipe/external/eigen_archive'),
-    join_paths(MEDIAPIPE_HOME, 'bazel-mediapipe/external/com_google_absl'),
+    join_paths(MEDIAPIPE_HOME, 'bazel-mediapipe', 'external', 'eigen_archive'),
+    join_paths(MEDIAPIPE_HOME, 'bazel-mediapipe', 'external', 'com_google_absl'),
   )
 
-  run_command('sh', '-c', meson.project_source_root() + '/tools/development/gen_mediapipe_libs.sh ' + nnstreamer_libdir, check : true)
+  run_command('sh', '-c', join_paths(meson.project_source_root(), 'tools', 'development', 'gen_mediapipe_libs.sh'), nnstreamer_libdir, check: true)
```

Comment:
nnstreamer_libdir may be not entered as a parameter for gen_mediapipe_libs.sh here.